deploy/databases-development.json

```diff
@@ -3,10 +3,12 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "databaseAccountName": {
-            "type": "string"
+            "type": "string",
+            "defaultValue": ""
```

Comment:
doing this will mask errors: if there is no defaultValue set, ARM will error if someone forgets to provide a value.

Reply:
OK, removing. By experiment, I got the impression that there have to be some value provided or it will hang. CLI is waiting for an input.

It is good to know, that SDK is failing loud.